Header header 
float64 latitude 
float64 longitude 
float64 altitude 
float64 utm_easting 
float64 utm_northing 
string   LatitudeDir
string   LongitudeDir
float64 UTC
uint8 zone 
string letter 
float64 hdop 
string gpgga_read
